src/applications/gi/components/profile/EstimatedBenefits.jsx

```diff
@@ -2,7 +2,21 @@ import React from 'react';
 import classNames from 'classnames';
 import { createId } from '../../utils/helpers';
 
-const CalculatorResultRow = ({ id, label, value, header, bold, visible }) =>
+const month = <span className="sr-only">per month</span>;
+const year = <span className="sr-only"> per year</span>;
+const ariaHiddenYear = <span aria-hidden="true">/yr</span>;
+const ariaHiddenMonth = <span aria-hidden="true">/mo</span>;
+
+const CalculatorResultRow = ({
+  id,
+  label,
+  value,
+  header,
+  bold,
+  visible,
+  ScreenReaderText,
+  ariaHidden,
```

Comment:
why not combine this with `ScreenReaderText` and wrap in a `<React.Fragment>` since you are providing both of these at same time so far?